--- a/build/PfpConverter.cs
+++ b/build/PfpConverter.cs
@@ -75,7 +75,7 @@
 			command3.Cmd = ShowFormCommand;
 			list4.Add(command3);
 			string cmdname = (args.Length < 1) ? "" : args[0];
-			Command command4 = list.Find((Command x) => x.Name.Equals(cmdname));
+			Command command4 = list.Find(delegate(Command x) { return x.Name.Equals(cmdname); });
 			if (command4 == null)
 			{
 				PrintHelp(list);
@@ -639,7 +639,7 @@
 	public class OptionForm : Form
 	{
 		[CompilerGenerated]
-		private string <MasterPassword>k__BackingField;
+		private string _MasterPassword_k__BackingField;
 
 		private Button formButtonOk;
 
@@ -654,12 +654,12 @@
 			[CompilerGenerated]
 			get
 			{
-				return <MasterPassword>k__BackingField;
+				return _MasterPassword_k__BackingField;
 			}
 			[CompilerGenerated]
 			private set
 			{
-				<MasterPassword>k__BackingField = value;
+				_MasterPassword_k__BackingField = value;
 			}
 		}
 
